bxin/ptg2m1m3/mountTelGenerator.ipynb

```diff
@@ -144,13 +174,52 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "#for setting constant elevation and azimuth angles.\n",
     "start_time = Time(datetime.now())\n",
     "freq = 1 #Hz\n",
     "end_time = start_time + timedelta(minutes=80)\n",
     "while Time(datetime.now())< end_time:\n",
     "    await asyncio.sleep(1.0/freq)\n",
     "    #mountCtrl.tel_elevation.set_put(angleActual=mountAngle.angleActual+dAngle)\n",
-    "    mountCtrl.tel_elevation.set_put(angleActual=89.5)"
+    "    #mountCtrl.tel_elevation.set_put(angleActual=89.5)\n",
+    "    #mountCtrl.tel_elevation.set_put(actualPosition=75)\n",
+    "    mountCtrl.tel_elevation.set_put(actualPosition=88)\n",
+    "    mountCtrl.tel_azimuth.set_put(actualPosition=45.0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#this cell is used when we do not know how much we can deviate \n",
+    "#from the real elevation angle (as measured by the inclinometer)\n",
+    "# we change the angle as a function of time, and stop at round numbers\n",
+    "\n",
+    "freq = 1 #Hz\n",
+    "startAngle = 86\n",
+    "stopAngle = 75\n",
```

Comment:
75 will kill the M2. Maybe comment the available value is 82 degree when the M2 inclinometer is ~89.6 degree.

Reply:
That is intended. The user won't know where to stop before starting this notebook. I imagine this will be useful when we re-test M1M3.

Reply:
The user is also expected to change the 75 to any desired value before this cell is run.